kiwi/xml_description.py

```diff
@@ -86,17 +95,19 @@ def load(self):
                 '%s: %s' % (type(e).__name__, format(e))
             )
         if not validation_ok:
-            raise KiwiDescriptionInvalid(
-                'Schema validation for %s failed' % self.description
-            )
+            if self.description:
+                message = 'Schema validation for %s failed' % self.description
```

Comment:
Nitpick of formatting: lately we use in Python `.format(...)` instead of this way. Example:
```python
message = 'Schema validation of {0} had failed'.format(self.description)
```
Or with parameters:
```python
message = 'Schema validation of {descr} had failed'.format(descr=self.description)
```
JFYI.